apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -30,38 +30,59 @@
 
 #define WM_TEST_COUNT  10
 
-#define USAGE															\
-	"\n usage: wm_test [options]\n"										\
-	"\n run Wi-Fi Manager:\n"											\
-	"	 wm_test start(default: station mode)\n"						\
-	"	 wm_test stop\n"												\
-	"	 wm_test stats\n"                                               \
-	"\n softap mode options:\n"											\
-	"	 wm_test softap [ssid] [password]\n"							\
-	"\n station mode options:\n"										\
-	"	 wm_test sta\n"													\
-	"	 wm_test join [ssid] [security mode] [password]\n"				\
-	"	    (1) [security mode] is optional if not open mode\n"			\
-	"	    (2) [password] is unnecessary in case of open mode\n"		\
-	"	 wm_test leave\n"												\
-	"	 wm_test cancel\n"												\
-	"\n run scan:\n"													\
-	"	 wm_test scan\n"												\
-	"\n get current state:\n"											\
-	"	 wm_test mode\n\n"												\
-	"\n set a profile:\n"											\
-	"	 wm_test set [ssid] [security mode] [password]\n"			\
-	"	 security mode examples : open, wep_shared \n"				\
-	"	               wpa_aes, wpa_tkip, wpa_mixed  \n"			\
-	"	               wpa2_aes, wpa2_tkip, wpa2_mixed  \n"			\
-	"	               wpa12_aes, wpa12_tkip, wpa12_mixed  \n"		\
-	"	               (*_ent for enterprise)  \n"					\
-	"\n get a profile:\n"											\
-	"	 wm_test get\n"												\
-	"\n remove a profile:\n"										\
-	"	 wm_test reset\n\n"											\
-	"\n repeat test of APIs:\n"										\
-	"	 wm_test auto [softap ssid] [softap password] [ssid] [security mode] [password]\n\n" \
+#define SOFTAP_USAGE							\
+	"\n softap mode options:\n"					\
+	"	 wm_test softap [ssid] [password]\n"
+
+#define STA_USAGE													\
+	"\n station mode options:\n"									\
+	"	 wm_test sta\n"												\
+	"	 wm_test join [ssid] [security mode] [password]\n"			\
+	"	    (1) [security mode] is optional if not open mode\n"		\
+	"	    (2) [password] is unnecessary in case of open mode\n"	\
+	"	 wm_test leave\n"											\
+	"	 wm_test cancel\n"
+
+#define SCAN_USAGE								\
+	"\n run scan:\n"							\
+	"	 wm_test scan\n"
+
+#define PROFILE_USAGE											\
+	"\n set a profile:\n"										\
+	"	 wm_test set [ssid] [security mode] [password]\n"		\
+	"	 security mode examples : open, wep_shared \n"			\
+	"	               wpa_aes, wpa_tkip, wpa_mixed  \n"		\
+	"	               wpa2_aes, wpa2_tkip, wpa2_mixed  \n"		\
+	"	               wpa12_aes, wpa12_tkip, wpa12_mixed  \n"	\
+	"	               (*_ent for enterprise)  \n"				\
+	"\n get a profile:\n"										\
+	"	 wm_test get\n"											\
+	"\n remove a profile:\n"									\
+	"	 wm_test reset\n\n"
```

Comment:
I think 'reset' is not good for remove a profile

Reply:
your comment isn't related to current PR. I'll fix it next PR.